### PR TITLE
FDR-297: Adding okta for enterprise

### PR DESCRIPTION
**Pull request by jamescgarrett**
### Changes

- Adding `okta` to the enterprise strategy list.
- Adding CSS for `okta`
- Handling quick auth for `okta`

HDR (default enterprise flow):
Screen Shot 2022-09-13 at 11 02 25 AM

Quick Auth:
Screen Shot 2022-09-13 at 11 44 38 AM

### References
https://auth0team.atlassian.net/browse/FDR-297

### Testing

* [ ] This change adds unit test coverage
* [ ] This change adds integration test coverage
* [x] This change has been tested on the latest version of the platform/language

### Checklist

* [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
* [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
* [x] All code quality tools/guidelines have been run/followed
* [x] All relevant assets have been compiled

### CODE REVIEW

File: src/connection/enterprise/quick_auth_screen.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { authButtonsTheme } from '../../connection/social/index';
 
 // TODO: handle this from CSS
 function icon(strategy) {
-  if (strategy === 'google-apps') return strategy;
+  if (strategy === 'google-apps' || strategy === 'okta') return strategy;
```
Comment (adrian-prananda) [inline src/connection/enterprise/quick_auth_screen.jsx:12]: I am assuming this is for enterprise? The other enterprise connections only return the 'auth0' icon (including PingFederate).

**Review by adrian-prananda — DISMISSED** [review]

File: src/connection/enterprise/quick_auth_screen.jsx (hunk 1 above)
Comment (jamescgarrett) [reply src/connection/enterprise/quick_auth_screen.jsx:12]: Yea, this is for enterprise. I was slightly confused by this section, so hoping  @auth0/dx-sdks-engineer can clarify if this is necessary

Comment (albertburchi-okta) [reply src/connection/enterprise/quick_auth_screen.jsx:12]: Just for my understanding - This means the avatar/icon only displays for Okta and Google right? And the other if is for microsoft and everything else displays default auth0 button?

**Review by albertburchi-okta — DISMISSED** [review]

File: src/connection/enterprise/quick_auth_screen.jsx (hunk 1 above)
Comment (stevehobbsdev) [reply src/connection/enterprise/quick_auth_screen.jsx:12]: That's correct. I'm unaware as to the history as to why it's like that though.

**Review by stevehobbsdev — APPROVED** [review]

**Comment by jamescgarrett** [comment]
@stevehobbsdev - Is something wrong with the BrowserStack test?

**Comment by stevehobbsdev** [comment]
@jamescgarrett The BrowserStack tests won't run successfully as this PR has been created from a fork. Our CI configuration prevents secrets from being available when building forks and so the BS tests won't be able to authenticate with BS.

I've verified them on my own machine against BS and we're all good.

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**